jack.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->  
+<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->  
+<!--[if !IE]><!--> <html lang="en"> <!--<![endif]-->  
+<head>
+    <title>Jack IJmker - Profiel</title>
+    <!-- Meta -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="mStack profiel van Jack IJmker">
+    <meta name="author" content="Xiaoying Riley at 3rd Wave Media">        
+    <link rel="icon" href="./assets/images/mStack-favicon.png" type="image/png">  
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,400italic,300italic,300,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
+    <!-- Global CSS -->
+    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">   
+    <!-- Plugins CSS -->
+    <link rel="stylesheet" href="assets/plugins/font-awesome/css/font-awesome.css">
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Heebo:400,700,900|Open+Sans" rel="stylesheet">
+    
+    <!-- Theme CSS -->  
+    <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+</head> 
+
+<body>
+    <div class="wrapper">
+        <div class="sidebar-wrapper">
+            <div class="profile-container">
+                <img class="profile" src="assets/images/jack.png" alt="Jack IJmker" />
```

Comment:
Fotootje zit niet in de commit, die moet je nog even expliciet toevoegen met git add.

Reply:
Hi Jasper.

ik kan de foto niet uploaden.... ik zie melding Uploads are disabled.
File uploads require push access to this repository.

Reply:
Foto heb ik inmiddels er bij ;) dus voorgaande bericht is niet meer van toepassing.